relaynet-jvm/src/main/java/tech/relaycorp/relaynet/CargoRelayClient.kt

```diff
@@ -0,0 +1,18 @@
+package tech.relaycorp.relaynet
+
+import kotlinx.coroutines.flow.Flow
+
+interface CargoRelayClient {
```

Comment:
I'd prefer to keep this outside the core library for two reasons:

- I'm not sure we'll ever implement an alternative binding to CogRPC (e.g., CoWebSockets).
- If we or someone else implements an alternative binding, I'm not sure whether we should try to require a particular interface. It's hard to anticipate that until we have a second binding we'd like to implement.

So for the time being, I'd suggest not to have this interface.